infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/OptionalSerializableFieldDefinition.java

```diff
@@ -45,8 +45,14 @@ public void writeField(final TObject source, final JsonGenerator gen) throws IOE
 
   @Override
   public void writeOpenApiField(final JsonGenerator gen) throws IOException {
-    gen.writeFieldName(name);
+    gen.writeObjectFieldStart(name);
+    gen.writeArrayFieldStart("allOf");
     type.serializeOpenApiTypeOrReference(gen);
+    gen.writeStartObject();
+    gen.writeBooleanField("required", false);
```

Comment:
`required:true` is probably more important than required:false, that flags things as not optional, and would put a red `*` in the openapi...

Reply:
Right well that's actually easier - we can just explicitly list the required fields in each object, so have switched over to doing it that way.